src/CommunityToolkit.Maui.Core/AssemblyInfo.shared.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("CommunityToolkit.Maui")]
```

Comment:
Let's move it to csproj

Reply:
Is there any difference?

Reply:
No, please remove this and delete this file.

If something in the `CommunityToolkit.Maui` project needs something from `CommunityToolkit.Maui.Core`, that API should be public. There should not be anything internal in `CommunityToolkit.Maui.Core` that is needed by `CommunityToolkit.Maui`.

```suggestion

```